app/config/parameters.yml.dist

```diff
@@ -11,11 +11,11 @@ parameters:
     coffee.node: ''
 
     database_driver: pdo_sqlite
-    database_host:
-    database_port:
-    database_name:
-    database_user:
-    database_password:
+    database_host: localhost
+    database_port: 3306
```

Comment:
+1 - have a config file template for parameters.yml in the place where the other config file templates for vagrant live